qiskit/visualization/timeline/generators.py

```diff
@@ -0,0 +1,470 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+A collection of functions that generate drawing objects from formatted input data.
+See :py:mod:`~qiskit.visualization.timeline.types` for more info on the required data.
+
+An end-user can write arbitrary functions that generate custom drawing objects.
+Generators in this module are called with the `formatter` kwarg. This data provides
+the stylesheet configuration.
```

Comment:
the `formatter` kwarg of what method?

Reply:
`formatter` is provided by core function, where this generator is called, residing in the PR 3/3. This is stylesheet `formatter` field so that user can update the appearance (mostly style of the generated object, i.e. object color, linewidth, etc...) by overwriting the stylesheet.